pallets/parachain-staking/src/inflation.rs

```diff
@@ -20,13 +20,10 @@
 use crate::{pallet::Config, types::BalanceOf};
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
-use sp_runtime::{traits::Saturating, Perquintill, RuntimeDebug};
-
-#[cfg(feature = "std")]
 use serde::{Deserialize, Serialize};
+use sp_runtime::{traits::Saturating, Perquintill, RuntimeDebug};
 
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
why do we remove this here?

Reply:
Serialization and deserialization are now required in the no_std environment. The genesis_config macro has been updated, as seen here: https://github.com/paritytech/substrate/pull/14306/files#diff-f6aee1a6b4a03b3f4366510598ac506d66658b747049868c583dda3863c484ad